pkg/ebpf/common.go

```diff
@@ -1,28 +1,30 @@
 package ebpf
 
 import (
+	"bufio"
 	"encoding/binary"
-	"errors"
 	"fmt"
+	"os"
+	"path"
 	"strings"
 	"unsafe"
 
 	"github.com/DataDog/datadog-agent/pkg/process/util"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/pkg/errors"
 )
 
-var (
-	// Feature versions sourced from: https://github.com/iovisor/bcc/blob/master/docs/kernel-versions.md
-	// Minimum kernel version -> max(3.15 - eBPF,
-	//                               3.18 - tables/maps,
-	//                               4.1 - kprobes,
-	//                               4.4 - perf events)
-	// 	                      -> 4.4
-	minRequiredKernelCode = linuxKernelVersionCode(4, 4, 0)
-
-	// eBPF got backported to centOS
-	// see: https://www.redhat.com/en/blog/introduction-ebpf-red-hat-enterprise-linux-7
-	centOSMinRequiredKernelCode = linuxKernelVersionCode(3, 10, 0)
-)
+var requiredKernelFuncs = []string{
```

Comment:
Can we keep the comment:`// Feature versions sourced from: https://github.com/iovisor/bcc/blob/master/docs/kernel-versions.md`

At the top of this?

Reply:
👍